lib/cli_common/cli_common/pulse.py

```diff
@@ -45,8 +45,22 @@
         connection_global=False
     )
 
-    queue = 'queue/{}/{}'.format(user, exchange)
+    exchange_name = exchange
```

Comment:
Can you add some comments here, I had to read it a few times to understand all the possible branches.

Reply:
:+1: